pims/cine.py

```diff
@@ -668,3 +887,20 @@ def _sixteen2twelve(b):
         a[k+2:k2:3] = ((b1 & 0x0FF) << 0)
 
     return a
+
+
+def _remove_trailing_x00(dic):
+    """Remove binary null character b'\x00' in dictionary dic entries."""
+    for k, v in dic.items():
+        if isinstance(v, bytes):
+            try:
+                dic[k] = v.decode('utf8').replace('\x00', '')
```

Comment:
You don't need to `decode` if you want to use replace, you can just do `.replace(b'\x00', b'')`

Reply:
Ok, I simplified the code, thanks for the tip.

Reply:
I realize that my previous solution with `decode` outputs a `str`, which was the desired behavior. The solution with only `.replace` in commits [`e9b0a5f`](https://github.com/soft-matter/pims/pull/317/commits/e9b0a5fde880fe996bafc4b62275c8c107b908f3) and [`87e8e45`](https://github.com/soft-matter/pims/pull/317/commits/87e8e45ebf374d5e36ffae0a836e30d9b4cb8d25) returns a byte. Can we go back to the previous solution?

Reply:
@baneel It sounds like you should go back to your old solution with `decode` if the new one does in fact fail!

Reply:
@nkeim, which is what I've committed. Thanks!